molecule.yml

```diff
@@ -32,9 +32,12 @@ ansible:
 # configuration options for the internal call to vagrant
 vagrant:
   raw_config_args:
-    - "landrush.enabled = true"
-    - "landrush.tld = 'vm'"
-    - "landrush.guest_redirect_dns = false"
+    - "hostmanager.enabled = true"
```

Comment:
Have yo tried to use vagrant-hostsupdater instead of vagrant-hostmanager??
With vagrant-hostsupdater more code is not needed.

Reply:
I haven't, but I'll give a try

Reply:
I have tested and I think that cannot be used in this role. 

AFAIK vagrant-hostsupdater only creates entries to your **/etc/hosts** in the host. The problem is that we need to add entries in guests too because kafka cannot see zookeeper otherwise.